pigpen-cascading/src/main/java/pigpen/cascading/OperationUtil.java

```diff
@@ -13,6 +13,8 @@
 
 public class OperationUtil {
 
+  public static final String SENTINEL_VALUE = "39c63d213f5cba3c7";
```

Comment:
Maybe "pigpen.cascading.OperationUtil/SENTINEL_VALUE"? The idea is to find a value that the user would _never_ use. It's entirely a gut feeling, but it seems like random numbers could occur in user data, but this namespace should never be in their code. Something random is almost less avoidable than something very specific, but unlikely to be in their data.